src/osemgrep/reporting/Sarif_output.mli

```diff
@@ -1,5 +1,6 @@
 val sarif_output :
   bool ->
+  string ->
   Rule.hrules ->
   Semgrep_output_v1_t.cli_output ->
   Sarif.Sarif_v_2_1_0_t.sarif_json_schema
```

Comment:
What about some labels for the bool and string arguments + some brief documentation what this module is about.

Reply:
Done.